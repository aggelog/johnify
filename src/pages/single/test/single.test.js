import React from 'react';
import renderer from 'react-test-renderer';

import Single from '../single';

it('renders correctly', () => {
  const tree = renderer
    .create(<Single
      gif={{}}
      error={false}
      match={{params: { id: '1'}}}
      getSingleGif={jest.fn()}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
