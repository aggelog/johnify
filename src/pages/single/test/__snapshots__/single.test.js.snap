// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-md-12 col-lg-12 col-xl-12 single-page__title-section"
    >
      <h3 />
    </div>
    <div
      className="col-xs-12 col-md-12 col-lg-12 col-xl-12"
    >
      <div
        style={
          Object {
            "height": 100,
            "margin": "auto",
            "maxWidth": "100%",
            "width": 100,
          }
        }
      >
        <img
          alt="..."
          src=""
          style={
            Object {
              "maxWidth": "100%",
            }
          }
        />
      </div>
      <div
        className="single-page__action-bar"
      >
        <div
          className="SocialMediaShareButton SocialMediaShareButton--facebook social-buttons_container"
          onClick={[Function]}
          onKeyPress={[Function]}
          role="button"
          style={
            Object {
              "width": 32,
            }
          }
          tabIndex="0"
        >
          <div
            style={
              Object {
                "height": 32,
                "width": 32,
              }
            }
          >
            <svg
              className="social-icon social-icon--facebook "
              height={32}
              viewBox="0 0 64 64"
              width={32}
            >
              <g>
                <circle
                  cx="32"
                  cy="32"
                  fill="#3b5998"
                  r="31"
                  style={Object {}}
                />
              </g>
              <g>
                <path
                  d="M34.1,47V33.3h4.6l0.7-5.3h-5.3v-3.4c0-1.5,0.4-2.6,2.6-2.6l2.8,0v-4.8c-0.5-0.1-2.2-0.2-4.1-0.2 c-4.1,0-6.9,2.5-6.9,7V28H24v5.3h4.6V47H34.1z"
                  fill="white"
                />
              </g>
            </svg>
          </div>
        </div>
        <div
          className="SocialMediaShareButton SocialMediaShareButton--twitter social-buttons_container"
          onClick={[Function]}
          onKeyPress={[Function]}
          role="button"
          style={
            Object {
              "width": 32,
            }
          }
          tabIndex="0"
        >
          <div
            style={
              Object {
                "height": 32,
                "width": 32,
              }
            }
          >
            <svg
              className="social-icon social-icon--twitter "
              height={32}
              viewBox="0 0 64 64"
              width={32}
            >
              <g>
                <circle
                  cx="32"
                  cy="32"
                  fill="#00aced"
                  r="31"
                  style={Object {}}
                />
              </g>
              <g>
                <path
                  d="M48,22.1c-1.2,0.5-2.4,0.9-3.8,1c1.4-0.8,2.4-2.1,2.9-3.6c-1.3,0.8-2.7,1.3-4.2,1.6 C41.7,19.8,40,19,38.2,19c-3.6,0-6.6,2.9-6.6,6.6c0,0.5,0.1,1,0.2,1.5c-5.5-0.3-10.3-2.9-13.5-6.9c-0.6,1-0.9,2.1-0.9,3.3 c0,2.3,1.2,4.3,2.9,5.5c-1.1,0-2.1-0.3-3-0.8c0,0,0,0.1,0,0.1c0,3.2,2.3,5.8,5.3,6.4c-0.6,0.1-1.1,0.2-1.7,0.2c-0.4,0-0.8,0-1.2-0.1 c0.8,2.6,3.3,4.5,6.1,4.6c-2.2,1.8-5.1,2.8-8.2,2.8c-0.5,0-1.1,0-1.6-0.1c2.9,1.9,6.4,2.9,10.1,2.9c12.1,0,18.7-10,18.7-18.7 c0-0.3,0-0.6,0-0.8C46,24.5,47.1,23.4,48,22.1z"
                  fill="white"
                />
              </g>
            </svg>
          </div>
        </div>
        <div
          className="social-buttons_container"
          onClick={[Function]}
          style={
            Object {
              "width": 32,
            }
          }
        >
          <svg
            enableBackground="new 0 0 800 800"
            version="1.1"
            viewBox="0 0 800 800"
            xmlSpace="preserve"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              id="Layer_1"
            >
              <path
                d="M771.9,400c0,205.3-166.6,371.9-371.9,371.9S28.1,605.3,28.1,400S194.7,28.1,400,28.1   S771.9,194.7,771.9,400z"
                fill="#4DB6AC"
              />
            </g>
            <g
              id="Layer_2"
            >
              <g>
                <circle
                  cx="470.9"
                  cy="270.2"
                  fill="none"
                  r="70.6"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeMiterlimit="10"
                  strokeWidth="16"
                />
                <circle
                  cx="257.4"
                  cy="416.5"
                  fill="none"
                  r="70.6"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeMiterlimit="10"
                  strokeWidth="16"
                />
                <circle
                  cx="490.3"
                  cy="529.8"
                  fill="none"
                  r="70.6"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeMiterlimit="10"
                  strokeWidth="16"
                />
                <line
                  fill="none"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeMiterlimit="10"
                  strokeWidth="16"
                  x1="314.7"
                  x2="413.7"
                  y1="375.2"
                  y2="311.7"
                />
                <line
                  fill="none"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeMiterlimit="10"
                  strokeWidth="16"
                  x1="313.7"
                  x2="421.8"
                  y1="459.1"
                  y2="512.4"
                />
              </g>
            </g>
          </svg>
        </div>
        <a
          className="social-buttons_container"
          download=""
          href=""
          style={
            Object {
              "width": 32,
            }
          }
          target="_blank"
        >
          <svg
            height="32px"
            version="1.1"
            viewBox="0 0 24 24"
            width="32px"
          >
            <title />
            <desc />
            <defs />
            <g
              fill="none"
              fillRule="evenodd"
              id="miu"
              stroke="none"
              strokeWidth="1"
            >
              <g
                id="Artboard-1"
                transform="translate(-251.000000, -155.000000)"
              >
                <g
                  id="slice"
                  transform="translate(215.000000, 119.000000)"
                />
                <path
                  d="M263,170.081066 L259.964466,167.045532 L259.257359,167.752639 L263.5,171.99528 L267.742641,167.752639 L267.035534,167.045532 L263.998437,170.082629 L263.998437,161 L263,161 L263,170.081066 Z M252,166.5 C252,160.148725 257.148725,155 263.5,155 C269.851275,155 275,160.148725 275,166.5 C275,172.851275 269.851275,178 263.5,178 C257.148725,178 252,172.851275 252,166.5 Z"
                  fill="#000000"
                  id="circle-arrow_down-download-glyph"
                />
              </g>
            </g>
          </svg>
        </a>
      </div>
    </div>
  </div>
</div>
`;
